lib/python/src/bailo/helper/release.py

```diff
@@ -204,6 +217,9 @@ def download_all(self, path: str = os.getcwd(), include: list | str = None, excl
                 file for file in file_names if not any([fnmatch.fnmatch(file, pattern) for pattern in exclude])
             ]
 
+        logger.info(
+            f"Downloading {len(file_names)} of {len(orig_file_names)} files for version {str(self.version)} of {self.model_id}..."
+        )
```

Comment:
We can use f-strings with this although logging has the intended behaviour of formatting additional arguments into the log. We can use something like this. This applies to all the other places this is used.

```python
logger.info(
    f"Downloading %d of %d files for version %s of %s", len(file_names), len(orig_file_names), str(self.version), self.model_id
)
```

Reply:
Done